woocommerce/woocommerce-android
# Disable Custom amount section while its operation (addition, removal) is in progress

Closes: #10456 


### Description

This PR fixes crash when trying to edit custom amount while the custom amount deletion is in progress.

### Testing instructions

1. Navigate to the order creation screen
2. Click on Add custom amount row
3. Add custom amount
4. While the custom amount is being added, try clicking the custom amount row and notice nothing happens
5. Open the previosuly added custom amount and delete it
6. While the deletion is in progress, click on the custom amount section and notice nothing happens

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/1331230/ef67276b-369e-45f9-9675-e790b2ee76b5

https://github.com/woocommerce/woocommerce-android/assets/1331230/c753f2ad-e15a-404b-ae8e-960321bc606c
